src/fu-util.c

```diff
@@ -2136,18 +2137,33 @@ fu_util_download_metadata(FuUtilPrivate *priv, GError **error)
 		FwupdDevice *dev = g_ptr_array_index(devs, i);
 		if (fwupd_device_has_flag(dev, FWUPD_DEVICE_FLAG_SUPPORTED))
 			devices_supported_cnt++;
+		if (fwupd_device_has_flag(dev, FWUPD_DEVICE_FLAG_UPDATABLE))
+			devices_updatable_cnt++;
 	}
 
 	/* TRANSLATORS: success message -- where 'metadata' is information
 	 * about available firmware on the remote server */
 	g_string_append(str, _("Successfully downloaded new metadata: "));
 
-	g_string_append_printf(str,
-			       /* TRANSLATORS: how many local devices can expect updates now */
-			       ngettext("%u local device supported",
-					"%u local devices supported",
-					devices_supported_cnt),
-			       devices_supported_cnt);
+	if (devices_supported_cnt > 0) {
+		g_string_append_printf(
+		    str,
+		    /* TRANSLATORS: how many local devices can expect updates now */
+		    ngettext("%u local device supported",
+			     "%u local devices supported",
+			     devices_supported_cnt),
+		    devices_supported_cnt);
+	} else {
+		g_string_append_printf(
+		    str,
+		    /* TRANSLATORS: message to tell user not to expect updates from remote */
+		    ngettext("Fwupd can update %u device on this system but no firmware updates "
+			     "have been published to any remote servers",
+			     "Fwupd can update %u devices on this system but no firmware updates "
+			     "have been published to any remote servers",
```

Comment:
I don't think we refer to Fwupd anywhere else, and it's a bit verbose as a string tbh -- what about something like `%u local devices supported, with no published updates`